lib/cmr/urs_client.rb

```diff
@@ -98,5 +272,45 @@ def build_connection
         conn.basic_auth(ENV['urs_username'], ENV['urs_password'])
       end
     end
+
+    # make the search results match the structure of the cmr results
+    def reformat_search_results(results, page_num=1, page_size=25)
+      items = results.map do |item|
+        { 'group_id' => item['group_id'],
+          'name' => item['name'],
+          'description' => item['description'],
+          'provider_id' => item['tag']
+        }
+      end
+
+      index = 0
+      if (page_num)
+        lower = (page_num-1)*page_size
+        upper = (page_num)*page_size
+      else
+        lower = 0
+        upper = 1000000
+      end
+
+      items.each do |item|
+        if (index >= lower and index <= upper)
+          item['member_count'] = get_edl_group_member_count(item['group_id'])
```

Comment:
We need to write EDL improvement ticket so we don't have to make a separate call to get counts